app/uk/gov/hmrc/eusubsidycompliancefrontend/views/LeadAccountPage.scala.html

```diff
@@ -96,10 +87,17 @@
       <li>@messages(s"$key.ul1-li2")</li>
     </ul>
 
-    @govukDetails(Details(
-      summary = Text(messages(s"$key.details.title")),
-      content = HtmlContent(detailsHtml)
-    ))
+    <div id ="page-details-title">@H2(messages(s"$key.details.title"), Some("govuk-heading-m govuk-!-margin-top-6"))</div>
```

Comment:
H2 has an id parameter, no need to wrap it in a div

Reply:
Sorry I know ive just approved, but just seen - @ymabdulahi I don't think youve added the ID using the H2 Param, youve just removed the DIV.